docs/API.md

```diff
@@ -979,26 +979,26 @@ Example response:
 
 An object with the following properties:
 
-* `"username"`: An *username*. The name of the account of an authorized user.
-* `"api_key"`: An *api key*. The api_key associated with the username.
-* `"domains"`: A list of *domain names*. The domains to be tested.
+* `"username"`: An *username*, required. The name of the account of an authorized user.
+* `"api_key"`: An *api key*, required. The api_key associated with the username.
+* `"domains"`: A list of *domain names*, required. The domains to be tested.
 * `"test_params"`: As described below.
 
 The value of `"test_params"` is an object with the following properties:
 
-* `"client_id"`: A *client id*, optional.
-* `"profile"`: A *profile name*, optional.
-* `"client_version"`: A *client version*, optional.
-* `"nameservers"`: A list of *name server* objects, optional.
-* `"ds_info"`: A list of *DS info* objects, optional.
+* `"client_id"`: A *client id*, optional. (default: `Zonemaster Batch Scheduler`)
+* `"profile"`: A *profile name*, optional. (default: `default`)
+* `"client_version"`: A *client version*, optional. (default: `1.0`)
```

Comment:
"client_id" and "client_version" should be undef unless given by the client. There is no reason to have a default value.